### PR TITLE
Icons meaning

### PR DESCRIPTION
**Pull request by LiorGnieslawMoveo**

### CODE REVIEW

File: epicure-frontend/src/components/IconsMeaning/IconsMeaning.scss [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    .icons-title {
+        font-family: Helvetica Neue;
+        font-size: 18px;
+        font-weight: 200;
+        line-height: 35px;
+        letter-spacing: 1.25px;
+        @include desktop {
+            font-size: 30px;
+            letter-spacing: 2.140000104904175px;
```
Comment (MoranOshiaMoveo) [inline epicure-frontend/src/components/IconsMeaning/IconsMeaning.scss]: round it

File: epicure-frontend/src/components/IconsMeaning/IconsMeaning.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        }
+    }
+
+    .icons-container {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        padding: 39px 476px 39px 476px;
```
Comment (MoranOshiaMoveo) [inline epicure-frontend/src/components/IconsMeaning/IconsMeaning.scss]: why do u need such a  large padding, see how can u avoid it

File: epicure-frontend/src/components/IconsMeaning/IconsMeaning.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                display: flex;
+                flex-direction: column;
+                align-items: center;
+            }
+
+            img {
+                @include desktop {
+                    width: 77.42px;
+                    height: 60px;
+                    padding: 0 0.47107696533203125px 2.5554962158203125px 0;
```
Comment (MoranOshiaMoveo) [inline epicure-frontend/src/components/IconsMeaning/IconsMeaning.scss]: round

File: epicure-frontend/src/components/IconsMeaning/IconsMeaning.scss [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+                }
+            }
+
+            h3 {
+                font-family: Helvetica Neue;
+                font-size: 18px;
+                font-weight: 200;
+                line-height: 30px;
+                letter-spacing: 1.9700000286102295px;
```
Comment (MoranOshiaMoveo) [inline epicure-frontend/src/components/IconsMeaning/IconsMeaning.scss]: round it

File: epicure-frontend/src/components/IconsMeaning/IconsMeaning.scss [hunk 5]
```diff
@@ -0,0 +1,65 @@
+@import '../../styles/mixins.scss';
+
+.icons-meaning-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background: #FAFAFA;
```
Comment (MoranOshiaMoveo) [inline epicure-frontend/src/components/IconsMeaning/IconsMeaning.scss:8]: set as variables

**Review by MoranOshiaMoveo — CHANGES_REQUESTED** [review]

File: epicure-frontend/src/components/IconsMeaning/IconsMeaning.scss (hunk 5 above)
Comment (LiorGnieslawMoveo) [reply epicure-frontend/src/components/IconsMeaning/IconsMeaning.scss:8]: I've created a variables file in the prior branches so once I will merge them I will apply the styling on all the components - is that ok?

**Review by MoranOshiaMoveo — APPROVED** [review]

**Pull request merged**